applications/vanilla/library/class.categorycollection.php

```diff
@@ -461,6 +461,42 @@ public function getTree($parentID = -1, $options = []) {
         return $tree;
     }
 
+    /**
+     * Get the id's of a category's descendants.
+     *
+     * @param int $parentIDs
+     * @param array $options
+     * @return array
+     */
+    public function getDescendantIDs($parentIDs = -1, $options = []) {
+        $ids = [];
+        $parentIDs = [$parentIDs] ?: [-1];
+        $defaultOptions = [
+            'maxDepth' => 3,
+            'permission' => 'PermsDiscussionsView'
+        ];
+        $options = array_change_key_case($options) ?: [];
```

Comment:
What's this for? And again I don't think we'll ever had the fallback of the ternary.

Reply:
removed it